magda-scala-common/src/main/resources/common.conf

```diff
@@ -41,31 +41,42 @@ regionSources = {
 		url = "https://s3-ap-southeast-2.amazonaws.com/magda-files/SA4.geojson"
 		idField = "SA4_CODE11"
 		nameField = "SA4_NAME11"
```

Comment:
Looks like we need to format the whitespace in here so it's either all spaces or all tabs :(

Reply:
👍 Will update that

Reply:
Replaced all indentation now looks alright